File: frontend/lib/modelos/reunion.dart
class Reunion {
  final String? reuId;
  final String gruId;
  final String? reuFecha;
  final String? reuHora;
  final String? reuLocalizacion;
  final String? reuDescripcion;

  Reunion({
    this.reuId,
    required this.gruId,
    this.reuFecha,
    this.reuHora,
    this.reuLocalizacion,
    this.reuDescripcion,
  });

  factory Reunion.fromJson(Map<String, dynamic> json) => Reunion(
        reuId: json['reuId'],
        gruId: json['gruId'] ?? '',
        reuFecha: json['reuFecha'],
        reuHora: json['reuHora'],
        reuLocalizacion: json['reuLocalizacion'],
        reuDescripcion: json['reuDescripcion'],
      );

  String get etiqueta => '${reuDescripcion ?? "Reunión"} - ${reuFecha ?? ""}';
}

File: backend/src/main/java/co/gerard/grupoestudio/modelo/reunion/Reunion.java
package co.gerard.grupoestudio.modelo.reunion;

import io.quarkus.hibernate.orm.panache.PanacheEntityBase;
import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import java.time.LocalDateTime;
import java.util.UUID;

@Entity
@Table(name = "reuniones")
public class Reunion extends PanacheEntityBase {

    @Id
    @GeneratedValue(strategy = GenerationType.UUID)
    @Column(name = "reu_id")
    public UUID reuId;

    @Column(name = "gru_id", nullable = false)
    public UUID gruId;

    @Column(name = "reu_fecha", nullable = false)
    public java.sql.Date reuFecha;

    @Column(name = "reu_hora")
    public java.time.LocalTime reuHora;

    @Column(name = "reu_localizacion", length = 100)
    public String reuLocalizacion;

    @Column(name = "reu_descripcion")
    public String reuDescripcion;

    @Column(name = "reu_creacion")
    public LocalDateTime reuCreacion;

    // Método estático tipado
    public static Reunion findById(UUID id) {
        return find("reuId", id).firstResult();
    }
}
